Resources/Prototypes/DeltaV/Entities/Markers/warp_points.yml

```diff
@@ -0,0 +1,338 @@
+# Off station
+- type: entity
+  id: WarpPointEvacShuttle
+  parent: WarpPoint
+  suffix: Evac Shuttle
+  components:
+  - type: WarpPoint
+    location: warp-point-evac
+  - type: Sprite
+    state: pink
```

Comment:
why dupe this for every single warp point preset

Reply:
Good point

Reply:
point like warp point?!?!?!?!

Reply:
Deltanedas makes great warp points!